jbot/src/main/java/me/ramswaroop/jbot/core/facebook/models/Event.java

```diff
@@ -305,4 +305,22 @@ public String toString() {
             return null;
         }
     }
+
+    /**
+     * Match the pattern with different attributes based on the event type.
+     *
+     * @return the pattern string
+     */
+    public String getPatternFromEventType() {
```

Comment:
But There was a lot of coupling as the method was living in Bot class initially and using an instance of Event class. So it might be better to move the method and reduce the coupling.